main.py

```diff
@@ -1,10 +1,72 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main():
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Player.objects.all().delete()
+    Guild.objects.all().delete()
+
+    with open("players.json") as file:
+        players_data = json.load(file)
+
+    races = [players_data[element]["race"] for element in list(players_data)]
+    guilds = [players_data[element]["guild"] for element in list(players_data)]
+
+    for race in races:
```

Comment:
Also, you can use get_or_create() method to create object if does not exist(just for info)